tests/components/netatmo/test_init.py

```diff
@@ -422,11 +415,17 @@ async def fake_post(*args, **kwargs):
         )
         await hass.async_block_till_done()
 
+        mock_addwebhook.assert_called_once()
+        mock_dropwebhook.assert_not_awaited()
+
         async_fire_time_changed(
             hass,
-            dt.utcnow() + timedelta(seconds=60),
+            dt.utcnow() + timedelta(seconds=60000),
```

Comment:
What happens after 60000 seconds?

Reply:
This was a dilettantish attempt  to trigger `async_get_access_token` from `api.py`. I reverted it to a more sane number.